### ISSUE TITLE
Respect Ubooquity 2.x.x ReverseProxy default setting

### ISSUE THREAD

**Issue by RaEyE** [sethchhim/Kuboo#36]
### Description
Since Ubooquity 2 the Sever pays respect to the use of reverseProxy.
Therefore the defaultsetting of the Server expects requests to the Server on <ServerIP>:2202/uboquity/.
This also has an effect on the opds streams.

### Proposed solution
Therefore the standard setting in the Kuboo Android-App should also pay respect to this.

### Systeminformation
ServerOS: unRAID v6.3.5
ubooquity Version: 2.1.1
ubooquity Implementation: Docker (https://hub.docker.com/r/linuxserver/ubooquity/)

### Issue request type

[ ] Question  
[X] Enhancement  
[ ] Bug

**Comment by sethchhim** [comment]
Does this break your use case? I don't use a reverse proxy but if you are willing to set up a mock testing environment for me, I will be happy to test it. sethchhim@gmail.com

**Comment by RaEyE** [comment]
Hello,

no, it does not break my use case.
I only thought about it while setting up a new installment of Ubooquity and first time using your app.
Only since the expectation of the ubooquity server includes a reverse proxy behaviour, I thought it might be appropriate to respect this by modifing the default values of Kuboo. Thats all.

A simple change in Kuboo or deletion of the reverse Proxy in ubooquity solves the "probblem" immediately.

TL/DR; An unexperienced user might haggle with this, therefore a change from http://<ServerIP>:2202/opds-comics to http://<ServerIP>:2202/<reverseProxyName>/opds-comics or a note explaning this behaviour would be welcome.
